integrationTests/vm/arwen/testdata/erc20-c-03/wrc20_arwen.c

```diff
@@ -78,69 +83,57 @@ void saveLogWith3Topics(byte *topic1, byte *topic2, byte *topic3, bigInt value)
 // is called immediately after the contract is created
 // will set the fixed global token supply and give all the supply to the creator
 void init() {
-  if (getNumArguments() != 1) {
-    char message[] = "wrong args num";
-    signalError(message, 14);
-    return;
-  }
+  CHECK_NUM_ARGS(1);
+  CHECK_NOT_PAYABLE();
 
   getCaller(sender);
   bigInt totalSupply = bigIntNew(0);
-  bigIntGetUnsignedArgument(0, totalSupply);
+  bigIntGetSignedArgument(0, totalSupply);
```

Comment:
Shouldn't this be unsigned?

Reply:
Well, the initial supply has to be a positive number. But it is handled as a signed number in the contract. @andrei-marinica?